src/shared/src/MathUtils.ts

```diff
@@ -18,4 +18,15 @@ export class MathUtils {
 
        return (Math.round(val * coefficient) / coefficient).toString();
    }
+
+   public static diffAngle(a: number, b: number): number {
```

Comment:
/**
  * @returns difference between angle b and angle a, normalised to (-180, 180]
  */